source/lib/MediaInsightsEngineAPIHelper/MediaInsightsEngineAPIHelper/__init__.py

```diff
@@ -0,0 +1,103 @@
+import requests
+import json
+import os
+import logging
+import boto3
+import re
+import urllib3
+from requests_aws4auth import AWS4Auth
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+def env_variables():
+    try:
+        env_vars = {
+            'REGION': os.environ['MIE_REGION'],
+            'MIE_STACK_NAME': os.environ['MIE_STACK_NAME'],
+            'ACCESS_KEY': os.environ['AWS_ACCESS_KEY_ID'],
+            'SECRET_KEY': os.environ['AWS_SECRET_ACCESS_KEY']
+            }
+    except KeyError as e:
+        logging.error("ERROR: Missing a required environment variable: {variable}".format(variable=e))
+        raise Exception(e)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using `Exception` and `BaseException` can make your code prone to errors and difficult to maintain. Instead, we recommend using one of the [Built-in Exceptions](https://docs.python.org/3/library/exceptions.html) or creating a custom exception class that is derived from `Exception` or one of its subclasses.